src/components/Header/ShoppingCart.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const Img = styled.img`
+  cursor: pointer;
+  padding-top: 20px;
+`;
+
+ShoppingCart.propTypes = {
+  logo: PropTypes.string.isRequired,
+  width: PropTypes.number,
+  className: PropTypes.string,
+  alt: PropTypes.string,
+};
+
+ShoppingCart.defaultProps = {
+  width: 50,
+  className: "shopping-cart-icon",
+  alt: "Shopping Cart",
+};
+
+function ShoppingCart(props) {
+  const { logo, width, className, alt } = props;
+  return (
+    <Img className={className} src={logo} alt={alt} style={{ width: width }} />
```

Comment:
I don't feel these component is doing much in terms of usability. Looks like it has only one purpose: to render the `ShoppingCart` icon. Also, I'm seeing that you have a similar one for `StoreLogo` and both have very similar code. I'd suggest you to find a middle point where you can reuse one or the other to render `logo`. This way you will have one component that can be re used.